core/templates/pages/exploration-player-page/services/content-translation-manager.service.spec.ts

```diff
@@ -322,6 +324,14 @@ describe('Content translation manager service', () => {
       writtenTranslations.addWrittenTranslation(
         'ca_1', 'fr', 'unicode', 'fr 2');
 
+      // This throws "Type '"DummyInteraction"' is not assignable to type
+      // '"ImageClickInput" | "NumberWithUnits" | "NumericExpressionInput" |
+      // "NumericInput" | "DragAndDropSortInput" | "ItemSelectionInput" |
+      // "Continue" | "GraphInput" | "EndExploration" | ... 11 more ... |
+      // "RatioExpressionInput"'." We need to suppress this error because
+      // we need to change the value of 'interaction.id' for testing
+      // purposes.
+      // @ts-expect-error
       interaction.id = 'DummyInteraction';
```

Comment:
is there any issue with having a correct interactionId here?

Reply:
yes, as I defined the type definition for interactionID, it can no longer be assigned to a string type that is not in the key of InteractionSpecsConstants.INTERACTION_SPECS. So, this statement will throw an error if we try to assign 'DummyInteraction'.

Reply:
>  this is a unit test for custom Interaction_specs cases, so we cannot add correct interactionID. 

I'm not sure why we can't add a correct interactionID here? Can you explain this a bit more? :)

Reply:
As the test is intended to test the custom interaction, so, I think we don't need this test now but when I removed it, the frontend test coverage was reduced. So I didn't remove the test and suppress this error. I may have misinterpreted the test. Can you correct me if I'm wrong?

Reply:
After giving a deep look at this test, I am concluding that the test is intended to test customized args on the interaction_specs. So the correct interaction id can be assigned with customized args it to. Do I conclude it right?

Reply:
Thanks for looking into it @17sushmita. 

>Please confirm if I conclude it right?

I also think that's correct, I would suggest adding a correct interactionId!

Reply:
Okay, thanks for the guidance @DubeySandeep . I'm making a commit with all required changes.